sdk/data/azcosmos/cosmos_global_endpoint_manager.go

```diff
@@ -115,19 +117,39 @@ func (gem *globalEndpointManager) GetAccountProperties(ctx context.Context) (acc
 		return accountProperties{}, fmt.Errorf("failed to generate path for name-based request: %v", err)
 	}
 
-	ctx, cancel := context.WithTimeout(ctx, 60*time.Second)
-	azResponse, err := gem.client.sendGetRequest(path, ctx, operationContext, nil, nil)
-	cancel()
+	finalURL := gem.clientEndpoint
+	if path != "" {
+		finalURL = azruntime.JoinPaths(gem.clientEndpoint, path)
+	}
+
+	ctxt, cancel := context.WithTimeout(ctx, 60*time.Second)
+	defer cancel()
+	req, err := azruntime.NewRequest(ctxt, http.MethodGet, finalURL)
 	if err != nil {
-		return accountProperties{}, fmt.Errorf("failed to retrieve account properties: %v", err)
+		return accountProperties{}, err
 	}
 
-	properties, err := newAccountProperties(azResponse)
+	req.Raw().Header.Set(headerXmsDate, time.Now().UTC().Format(http.TimeFormat))
+	req.Raw().Header.Set(headerXmsVersion, "2020-11-05")
```

Comment:
nit: Do we have any const somewhere with this value?

Reply:
like supportedCapabilitiesHeaderValue

Reply:
I don't believe we do since I grabbed this code as-is from our client code:
<img width="220" alt="image" src="https://github.com/Azure/azure-sdk-for-go/assets/30335873/72887143-d671-432c-89b6-ce8c079c1368">
I could add it under the `version` file as a separate constant, or make a separate PR to add a constants file to the SDK which we can build on as it develops

Reply:
Add it to th cosmos-client.go file as a private const, and reuse it.